src/utils/utils.js

```diff
@@ -1,12 +1,15 @@
+const _ = require('lodash');
+
 const { execFile } = require('../../server/src/constants');
+const { isDevEnv, getDevQtumExecPath } = require('../../server/src/utils/utils');
```

Comment:
I feel a little skeptical about `utils/utils`, we should change the `utils.js` to `index.js` under utils folder.

Reply:
np, fixed!